scripts/tw-make

```diff
@@ -118,9 +125,12 @@ if __name__ == "__main__":
         level = int(level.lstrip("level"))
         make_game = textworld.challenges.CHALLENGES[challenge]
         game = make_game(level, options)
-        game_file = textworld.generator.compile_game(game, args.output, force_recompile=args.force)
+        game_file = textworld.generator.compile_game(game, args.output, force_recompile=args.force,
+                                                     file_ext="." + args.format)
```

Comment:
Could reuse `options.file_ext` right?  (Maybe `compile_game()` should just take `GameOptions`, or even get it from the `Game`?)

Reply:
`Game` doesn't have `GameOptions` at the moment.